docs/concepts/vega-chain.md

```diff
@@ -4,76 +4,75 @@ title: Vega Chain
 
-Everyone participating in staking is rewarded for keeping the network running (and in the future can be penalised for not meeting the requirements of running the network). 
+Everyone participating in keeping the network secure, robust and reliable, including nominators, is **rewarded** for keeping the network running. Not meeting the requirements of running the network can lead to penalties, such as **rewards being withheld**.
 
-<!-- Penalties: 
-- Validators can lose their VEGA to the network if they don't meet the requirements or prove to be bad actors. The tokens are sent to the insurance pool. (link to insurance pool section when available) 
-- Nominators will lose future rewards
+Read more: [Rewards](/docs/concepts/vega-chain#rewards)
 
-### ***Further reading***
-Link to spec for staking and validator rewards when publicly available. (valpol)
--->
+Read more: [Penalties](/docs/concepts/vega-chain#penalties)
 
 ### VEGA token
-Vega uses the VEGA ERC20 token for governance, which includes nominating validators, and creating and voting on governance proposals.
+Vega uses the VEGA ERC20 token for governance, which includes nominating validators to run nodes, and creating and voting on governance proposals.
 
 A VEGA token (or fraction) can be either unassociated or associated with a Vega key:
 
 * **Unassociated**: The tokenholder is free to do what they want with the token, but cannot nominate a validator with it
 * **Associated**: The token is locked in the staking smart contract and can be used to nominate a validator. It must be unassociated to withdraw it
 
-**All tokens can be used for staking and voting** on [governance proposals](/docs/concepts/vega-protocol#governance). This includes tokens that are locked in the vesting contract. Tokens that are staked can be used to vote, and tokens used to vote can be staked.
+**All tokens can be used for staking and voting** on governance proposals. This includes tokens that are locked in the vesting contract. Tokens that are staked can be used to vote, and tokens used to vote can be staked.
+
+Read more: [Governance of Vega](/docs/concepts/vega-protocol#governance)
 
 :::info
 A user's VEGA tokens must first be associated with a Vega key before they can be used for governance and staking.
 :::
 
-### Staking on Vega 
-Vega networks use the ERC20 token VEGA for staking. Staking requires the combined action of associating VEGA tokens (or fractions of a token) to the Vega staking bridge contract; and using those token(s) to nominate one or more validators. 
+### Bridges used for staking
+Because VEGA is an ERC20 token, all actions regarding staking are initiated on Ethereum, rather than on the Vega protocol. This allows VEGA to be staked to a Vega public key without any action on the Vega network, and without putting the tokens under the control of the Vega network.
 
```

Comment:
Comment (non-blocking): I find this a little misleading; the linking/unlinking a token to a Vega key is initiated by ethereal, the choice of validators is happening on Vega. Also might be useful to add something along the line of "All governance voting and change of nominations happen exclusively on the Vega chain, thus the only Ethereum gas fees that occur are the process of associating tokens and transferring the rewards.

Reply:
I've included your suggestions, and reorganised the section slightly to (hopefully) remove anything misleading about the eth/vega interaction